hw/ip/spi_host/data/spi_host_testplan.hjson

```diff
@@ -144,7 +211,58 @@
               - Ensure reset is handled correctly
             '''
       milestone: V2
-      tests: ["spi_host_stress_all"]
+      tests: []
+    }
+  ]
+  covergroups: [
+    {
+      name: tx_fifo_overflow_cg
+      desc: '''
+            Collect coverage to verify that an attempt was made to overflow the TX FIFO
+            by attempting to write to a full FIFO
+            '''
+    }
+    {
+      name: rx_fifo_underflow_cg
+      desc: '''
+            Collect coverage to verify that an attempt was made to underflow the RX FIFO
+            by attempting to read from an empty FIFO
+            '''
+    }
+    {
+      name: long_commands_cg
+      desc: '''
+            Collect coverage to verify that both a read and write command longer than 4 bytes
+            was seen
+            '''
+    }
+    {
+      name: sck_cs_timing_cg
+      desc: '''
+            Collect coverage to verify that all possible relationships between sck and cs was seen
+            using the CSNLEAD, CSNTRAIL and CSNIDLE registers
+            '''
+    }
+    {
+      name: cpol_cpha_cg
+      desc: '''
+            Collect coverage to verify that all 4 combinations of CPOL and CPHA was seen
+            '''
+    }
+    {
+      name: cdc_cg
+      desc: '''
+            Collect coverage on the relationship between the core_clk and the spi_sck to verify that
+            both scenarios with both a slow spi_sck, a very fast spi_sck (2x core_clk)
+            and where the two clocks are very close has been tested
+            '''
+    }
+    {
+      name: unaligned_data_cg
+      desc: '''
+            Collect coverage the alignment of writes to the data window
+            to verify that all possible alignments was seen
+            '''
```

Comment:
based on the testplan, there are more coverpoints we can add, such as speed(single, dual, quad), pass through, clock divider (max/min frequency), endian setting, duplex(half and full ) etc

Reply:
sure my assumptions was any register setting is a given..
but let me add them all  to make it clear